ttnn/ttnn/operations/binary.py

```diff
@@ -246,6 +246,10 @@ def _golden_function(input_tensor_a, input_tensor_b, *args, **kwargs):
 
 logaddexp2 = ttnn.register_operation(golden_function=_golden_function)(ttnn._ttnn.operations.binary.logaddexp2)
 
+divide = ttnn.register_operation()(ttnn._ttnn.operations.binary.divide)
```

Comment:
Would be nice to have the golden function implemented for `divide` and `bias_gelu`. Can be done on a future PR

Reply:
added